project-management/src/main/java/life/qbic/projectmanagement/domain/model/measurement/NGSMeasurement.java

```diff
@@ -42,28 +46,74 @@ public class NGSMeasurement implements MeasurementMetadata {
   @Column(nullable = false)
   private ProjectId projectId;
 
-  @Embedded
+  @Column(name = "instrument", columnDefinition = "longtext CHECK (json_valid(`instrument`))")
   private OntologyTerm instrument;
 
   @Convert(converter = MeasurementCode.MeasurementCodeConverter.class)
   private MeasurementCode measurementCode;
-
-  @ElementCollection
-  @CollectionTable(name = "measurement_samples", joinColumns = @JoinColumn(name = "measurement_id"))
-  @Column(name = "measured_sample")
+  @Column(name = "registration")
```

Comment:
Probably should be something like registrationTime

Reply:
Good point!